tasks/install-from-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot
```

Comment:
Make sure the name definition for each task has a period at the end, to match the style from my other roles.